package.json

```
@@ -10,7 +10,7 @@
     "debug": "~2.6.9",
     "express": "~4.16.0",
     "http-errors": "~1.6.2",
-    "pug": "~2.0.4",
+    "pug": "~2. sdflkjsdlfkjsdlfkjsdlkfj
```

This would be easier to work with
```suggestion
    "pug": "~2.0.3",
```